homework3/task03/tests/test_filter.py

```diff
@@ -0,0 +1,130 @@
+from project_filter.filter import Filter, make_filter
+
+
+def test_apply_positive_even():
+    positive_even = Filter(
+        lambda a: a % 2 == 0, lambda a: a > 0, lambda a: isinstance(a, int)
+    )
+    assert positive_even.apply(range(10)) == [2, 4, 6, 8]
+
+
+def test_apply_sum_with_7_is_zero():
+    sum_with_7_is_zero = Filter(lambda a: a != 0, lambda a: a + 7 == 0)
+    assert sum_with_7_is_zero.apply([-7, 13, 6.8, 9, 0]) == [-7]
+
+
+def test_apply_no_data_to_filter():
```

Comment:
It looks like a two tests

test_empty_filter_returns_all_items
test_filter_of_empty_list_return_empty_list

Reply:
Yes, thanks, good names:)